packages/client/src/index.ts

```diff
@@ -27,16 +27,19 @@ export type {
   OsdkActionParameters,
   OsdkObject,
   PageResult,
+  Result,
   WhereClause,
 } from "@osdk/client.api";
+
+export { isOk } from "@osdk/client.api";
 export { PalantirApiError } from "@osdk/shared.net.errors";
 
 export type { Client } from "./Client.js";
 export { createClient } from "./createClient.js";
 export { createPlatformClient } from "./createPlatformClient.js";
 
 export { createAttachmentFromRid } from "./createAttachmentFromRid.js";
+export { createAttachmentUpload } from "./object/AttachmentUpload.js";
 
 export { ActionValidationError } from "./actions/ActionValidationError.js";
-export { isOk } from "./ResultOrError.js";
```

Comment:
So we originally exporting this which I don't think we're actually using anywhere? https://github.com/palantir/osdk-ts/blob/3ad243443517d98469ed47959beb5ae5612fc67f/packages/client/src/ResultOrError.ts#L21 changed it to use this: https://github.com/palantir/osdk-ts/blob/3ad243443517d98469ed47959beb5ae5612fc67f/packages/client.api/src/object/Result.ts#L21